data/css/all-aboard.css

```diff
@@ -284,3 +321,15 @@ a:focus {
     height: 0;
     opacity: 0;
 }
+
+/* Hide visually, but not to screen readers */
+.visually-hidden {
```

Comment:
@schalkneethling should this be denoted as visually-hidden? It seems to be showing up fine